apps/ftu/js/navigation.js

```diff
@@ -246,6 +249,16 @@ var Navigation = {
             sharePerformance.checked = req.result[settingName] || false;
           };
         }
+
+        // If it's a dogfooder, we don't want them to disable the metrics.
+        var dogfood = settings && settings.createLock().get(DOGFOODSETTING);
+        if (dogfood) {
+          dogfood.onsuccess = function() {
+            if(dogfood.result[DOGFOODSETTING] === '1') {
+              sharePerformance.setAttribute('disabled', 'true');
```

Comment:
what if is previously set as true, and the flag value is `0`? 
shouldn't we cover that too?